examples/config-builder/three_qb_system.py

```diff
@@ -0,0 +1,317 @@
+import numpy as np
+
+from qualang_tools.config.configuration import *
+from qualang_tools.config.components import *
+from qualang_tools.config.builder import ConfigBuilder
+
+cnot_coupler = ArbitraryWaveform(
+    "cnot_coupler",
+    (
+        2.0
+        * np.pi
+        * np.array(
```

Comment:
Err, I hate the way black handles this...
What are these numbers? We need to put here functions that create them and not the raw numbers

Reply:
the precise numbers are not important for this example, switching to random numbers